app/src/hooks/domain/usePermissions.ts

```diff
@@ -45,6 +45,8 @@ function usePermissions() {
 
             const isSuperUser = !isGuestUser && !!userMe?.is_superuser;
 
+            const isGlobalValidator = !isGuestUser && !!userMe?.is_global_validator;
```

Comment:
Is this role generic or specific to local units? 

Also I feel like this should be permission instead of role @thenav56 @tnagorra @samshara

Reply:
@frozenhelium this is specific to local units.